buildbot/buildbot/config_default.py

```diff
@@ -54,7 +58,13 @@
 # the synthesis step (or running an opaque Makefile), the latter of
 # which has to be really long because synthesis is so slow.
 COMPILE_TIMEOUT = 120
-SYNTHESIS_TIMEOUT = 3000
+SYNTHESIS_TIMEOUT = 9000
 
 # Default Xilinx target platform.
 DEFAULT_PLATFORM = 'zed'
+DEFAULT_TARGET = 'sw_emu'
+TOOLCHAIN = 'f1'
+
+S3_BUCKET = 'test-bucket-1025132741'
+S3_DCP = 'DCPs'
+S3_LOG = 'SDAccel_log'
```

Comment:
Some comments here might be nice, to describe what these options do! For example, `TOOLCHAIN` is a big, important switch that people should be aware of (and which options are available). And `DEFAULT_TARGET` only matters for F1 (i.e., SDAccel), right? Same with the S3 options?